spec.md

```diff
@@ -1402,6 +1408,122 @@ Applying integer decoding with a divisor of `100` to create an array of 32-bit f
 [ 1.00, 1.00, 1.00, 1.00, 0.50, 0.50 ]
 ```
 
+### Extra data
+
+The following are fields that are __not__ supplied by the *RCSB*, and are provided for applications to transfer non-molecular data along with the molecular data.
+
+The restrictions on each of the fields are __not__ enforced at decode-time and are simple conventions to help guide applications.
+
+| Name            | length-restrictions     | 
+|-----------------|-------------------------|
+| bondProperties  | length of numBonds      |
+| atomProperties  | length of numAtoms      |
+| groupProperties | length of numGroups     |
+| chainProperties | length of numChains     |
+| modelProperties | length of numModels     |
+| extraProperties | None                    |
+
+#### bondProperties
+
+*Optional field*
+
+*Type*: [Map](#types)
+
+*Description*: A field meant to store information on specific bonds.  Must have the same length as [numBonds](#numbonds).
+
+*Convention key:value pairs*
```

Comment:
A description of what is meant exactly by 'convention' would be good.

Reply:
I will also rephrase this, I think I will change this to `reserved` key:value pairs and add a little explanation at the of the `extra data` header.

The idea was to have some `key`: `value` pairs locked so that you could save things like camera positions/color schemes between applications. (ie everyone use `cameraOrientation` not `app1_camera_orientation` and `app2_camera_orientation`).